katsdpimager/preprocess.cpp

```diff
@@ -611,13 +603,12 @@ void visibility_collector<P>::add(
 template<int P>
 void visibility_collector<P>::close()
 {
-    compress();
```

Comment:
Is this not needed because `compress(channel)` in `add` already flushes things?

Reply:
Exactly. I could probably remove the `close` method entirely, but it's fairly harmless and maybe I'll need it again in future.